Contributors.md

```diff
@@ -81,9 +81,7 @@
   </a>
 </td>
 <!-- End of column-7 -->
-</tr>
```

Comment:
Awesome work @Jummate Thanks for your contribution. 👍🏾

Can you please add back the ```<tr>``` tag as it allows the cards to be stacked in a column of 7 on the next line.

Reply:
All right.

Reply:
Do you mean I should create a new row after the 7th contributor? And that means the table will contain rows of 7 columns?

Reply:
@Evavic44

Reply:
No, the ```<tr>``` tag is responsible for creating a new row, but you removed it in your PR. 

Anyways some updates have been pushed since your last edit. Can you pull those updates to fix the conflicts. 🙂 If you have any issues doing that, let me know

Reply:
@Evavic44 
Oh I see. I didn't know it was supposed to be an arrangement of 7 columns in perrow.
Meanwhile, the image below shows what I met in the Contribution.md file after cloning the repo and before I edited at all. I had a feeling something wasn't right. That was the reason I took a shot before I made the changes.


![image](https://user-images.githubusercontent.com/40699198/148045335-f7a53576-df0d-4d2e-ac21-2e8c2880392e.png)

Reply:
According to the image, I thought the person who updated 8th column was supposed to enclose it in the <tr> tag. You know, <td> should not stand alone as the person did in the image.

Reply:
Yeah, I can see that. So what you'll do is add all the profiles after `<!--column-7-->` inside a new `<tr></tr>` tag. 

Like so. 👇🏾

```
<tr>

 <!-- Start of column-9 -->
<td align="center">
  <a href="https://github.com/mikey247">
    <img
      src="https://avatars.githubusercontent.com/u/93191705?v=4"
      width="100px"
    />
    <br />
    <sub>Michael Umeokoli</sub>
  </a>
</td>
<!-- End of column-9 -->

</tr>
```

Reply:
All right. I will just pull the updates, edit and push again.

Reply:
Okay, I'll be looking out for it. 🙂🙌🏾